app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -185,119 +77,16 @@ public function addMessageWhenCapture($observer)
         $order = $payment->getOrder();
 
         $method = $payment->getMethod();
+        $message = '';
+
         if (strtolower($method) == Bolt_Boltpay_Model_Payment::METHOD_CODE) {
-            $message = $this->_addMagentoOrderIdToMessage($order->getIncrementId());
+            $message .= ($incrementId = $order->getIncrementId()) ? $this->boltHelper()->__('Magento Order ID: "%s".', $incrementId) : "";
             if (!empty($message)) {
                 $observer->getEvent()->getPayment()->setPreparedMessage($message);
             }
         }
     }
-
-    /**
-     * Updates the Magento's interpretation of the Bolt transaction status on order status change.
-     * Note: this transaction status is not necessarily same as order status on the Bolt server.
-     * bolt_transaction_status field keeps track of payment status only from the magento plugin's perspective.
-     *
-     * @param $observer
-     */
-    public function updateBoltTransactionStatus($observer)
```

Comment:
why can we remove this now?

Reply:
This is repeated, unnecessary code.  The webhook path handles this via the `Bolt_Boltpay_Model_Payment::handleTransactionUpdate` method.

https://github.com/BoltApp/bolt-magento1/blob/v2.0.0/mPR.15.order-success-processing/app/code/community/Bolt/Boltpay/Model/Payment.php#L426-L513